style.css

```diff
@@ -0,0 +1,935 @@
+  }
+}
+
+/* Medium devices (tablets, 768px and up) */
+@media (min-width: 768px) {
+  #subheader .subheader-container {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    width: 90%;
+    margin: auto;
+  }
+
+  .navbar-custom .nav-container {
+    width: 90%;
+    margin: auto;
+  }
+
+  .navbar-custom .navbar-nav-container {
+    display: flex !important;
+  }
+
+  .navbar-custom .navbar-toggler {
+    display: none !important;
+  }
+
+  .navbar-custom .navbar-brand {
+    text-align: left;
+  }
+
+  .navbar-custom .navbar-nav .nav-item a {
+    padding-right: 10px;
+    padding-left: 10px;
+  }
+
+  .banner-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  #banner-section .main-title {
+    width: 100%;
+    font-weight: 900;
+    letter-spacing: 2px;
+    line-height: 72px;
+    font-size: 3.8rem;
+    display: flex;
+  }
+
+  #banner-section .date-location-info p {
+    text-align: center;
+  }
+
+  #banner-section .sub-title {
+    font-size: 1.9rem;
+    font-weight: 400;
+    margin-bottom: 14px;
+  }
+
+  #banner-section .banner-description {
+    width: 100%;
+    display: flex;
+    text-align: left;
+  }
+
+  #banner-section .banner-container {
+    width: 90%;
+    margin: auto;
+  }
+
+  .navbar-custom .header-action-btn {
+    width: 160px;
+  }
+
+  #main-program-section .col {
+    flex: 1 0 100%;
+    margin-bottom: 15px;
+  }
+
+  #main-program-section .program-list {
+    padding: 10px;
+  }
+
+  #main-program-section .program-list .program-card .card-body {
+    padding: 17px 36px;
+  }
+
+  #main-program-section .program-list .program-card .card-body img {
+    width: 18%;
+  }
+
+  #speakers-section .speaker-card p {
+    font-size: 20px;
+    width: 90%;
+    margin-top: 10px;
+    margin-bottom: 0;
+  }
+
+  #main-program-section .program-list .program-card .card-body p {
+    text-align: left;
+  }
+
+  #main-program-section .program-list .program-card .card-body h5 {
+    text-align: left;
+    width: 40%;
+  }
+
+  #speakers-section .speaker-card span {
+    font-size: 18px;
+    font-style: italic;
+    width: 90%;
+    font-weight: 600;
+  }
+
+  .speaker-line {
+    display: block;
+    margin-top: 15px;
+  }
+
+  #main-program-section .program-container {
+    width: 90%;
+    margin: auto;
+  }
+
+  section#about-wrapper #main-program-section {
+    padding-bottom: 0;
+    margin-bottom: 4em;
+  }
+
+  section#about-wrapper #partners-section {
+    display: none;
+  }
+
+  section#about-wrapper footer {
+    margin-top: 0;
+    padding: 2em 0;
+    background-color: var(--darkColor);
+    color: #fff;
+  }
+}
+
+/* Large devices (desktops, 992px and up) */
+@media (min-width: 992px) {
+  #subheader .subheader-container {
+    width: 80%;
+    margin: auto;
+  }
+
+  .navbar-custom .nav-container {
+    width: 80%;
+    margin: auto;
+  }
+
+  #banner-section .banner-container {
+    width: 80%;
+    margin: auto;
+  }
+

```

Comment:
Good work correcting this part of your project. But there's a difference in the Behance design from yours

- [ ] Kindly add a white background so in this way will match Behance design

![image](https://user-images.githubusercontent.com/93630700/211019141-3559a183-adf1-4c6f-aa7a-ad56abd54b26.png)